lib/shopify_api/rest/base.rb

```diff
@@ -2,6 +2,7 @@
 # frozen_string_literal: true
 
 require "active_support/inflector"
+require 'active_support/all'
```

Comment:
![image](https://github.com/Shopify/shopify-api-ruby/assets/10522258/74a4a86d-9209-46ec-bfa3-a1a5b4f50e02)

We should explicitly require hash_with_indifferent_access from active_support and not `all`